OpenRA.Game/Platform.cs

```diff
@@ -221,8 +221,9 @@ public static string UnresolvePath(string path)
 			if (path.StartsWith(SupportDir, compare))
 				path = SupportDirPrefix + path.Substring(SupportDir.Length);
 
-			if (path.StartsWith(GameDir, compare))
-				path = "./" + path.Substring(GameDir.Length);
+			var modDir = GameDir.Replace("engine" + Path.DirectorySeparatorChar, "");
```

Comment:
Hardcoding `engine` here really doesn't look right. What exactly are you trying to fix here?

Reply:
It is string comparing `/home/matthias/Entwicklung/OpenHV/engine/mods/hv/bits` with `/home/matthias/Entwicklung/OpenHV/mods/hv/bits` which fails so it outputs the full path.

I don't know a better way to get the mod path at runtime.

Reply:
https://github.com/OpenRA/OpenRAModSDK/blob/master/mod.config#L120 is configurable so hardcoding this here sounds like a bad idea. Can't we get the mod path(s) from the `Engine.ModSearchPaths` argument?

Reply:
I tried a few things and hopefully came up with something smart and reliable.